src/Application/Cli/TravisAutoDeployCommand.php

```diff
@@ -58,13 +58,6 @@ protected function configure()
                 getcwd()
             )
             ->addOption(
-                'php-version',
-                null,
-                InputOption::VALUE_REQUIRED,
-                'Specify for which php version you want to deploy documentation, mainly to avoid multiple deploys',
-                '5.4'
-            )
```

Comment:
Why remove this option? Even if we don't use it I imagine it can be used by some users?